### ISSUE TITLE
Exception

### ISSUE THREAD

**Issue by sachinprabhu007** [careerjet/careerjet-api-client-python#1]
> except Exception as e:

 instead of  

> except Exception, e: 

in init file

**Comment by daverickdunn** [comment]
Is `except Exception, e:` valid for any version of Python?

**#Abandonware**

**Comment by aaronsteers** [comment]
Same issue here.

**Comment by davebulaval** [comment]
Yes version 2 https://docs.python.org/2/tutorial/errors.html#defining-clean-up-actions

**Comment by davebulaval** [comment]
I've built a simple fix since no response from the owner: 
https://github.com/davebulaval/careerjet-api/tree/3.2

PyPi: https://pypi.org/project/careerjet-api/3.2/
